packages/client/components/SnackbarMessageAction.tsx

```diff
@@ -3,23 +3,20 @@ import {SnackAction} from './Snackbar'
 import styled from '@emotion/styled'
 import PlainButton from './PlainButton/PlainButton'
 import {PALETTE} from '../styles/paletteV3'
-import {DECELERATE} from '../styles/animation'
+import {Radius} from '../types/constEnums'
 
 interface Props {
   action: SnackAction | null | undefined
 }
 
 const Action = styled(PlainButton)({
-  borderRadius: 2,
+  borderRadius: Radius.SNACKBAR,
   color: PALETTE.ROSE_500,
   fontSize: 14,
   fontWeight: 600,
   marginLeft: 8,
   padding: 8,
-  transition: `background 100ms ${DECELERATE}`,
-  ':hover,:focus,:active': {
-    backgroundColor: '#ffffff0e'
-  }
+  backgroundColor: '#ffffff0e'
```

Comment:
-1 I think the action button still need to react when I hover my cursor over it.

https://user-images.githubusercontent.com/466991/177758630-173c497e-1d5e-434e-9e58-ff80b1642c5e.mov

cc @enriquesanchez

Reply:
OK